ttml_test.go

```diff
@@ -68,3 +68,20 @@ func TestTTMLBreakLines(t *testing.T) {
 
 	assert.Equal(t, strings.TrimSpace(string(c)), strings.TrimSpace(w.String()))
 }
+
+func TestTTMLWithOptionsNoIndent(t *testing.T) {
```

Comment:
Can you rename this test to `TestWriteToTTMLWithIndentOption`?

Reply:
OK, will rename it later.

Reply:
Fixes: [commit](https://github.com/asticode/go-astisub/pull/117/commits/83ca26d9703cf904e3f6254cecd621c7cd9fbf1f)